exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/Metadata.java

```diff
@@ -526,6 +534,48 @@ private void writeFile(ParquetTableMetadataDirs parquetTableMetadataDirs, Path p
   }
 
   /**
+   * Serializer for ParquetPath. Writes the path relative to the root path
+   */
```

Comment:
Why compute the relative during serialization? A more common approach is to store the relative paths in our internal data structures and serialize that relative value.

Doing things this way is awkward: we need to define a serializer and deserializer unnecessarily.

Also, see comment later on backward compatibility.